cfrontend/PrintClight.ml

```diff
@@ -79,9 +79,9 @@ let rec expr p (prec, e) =
   | Econst_int(n, _) ->
       fprintf p "%ld" (camlint_of_coqint n)
   | Econst_float(f, _) ->
-      fprintf p "%.15F" (camlfloat_of_coqfloat f)
+     fprintf p "%.18g" (camlfloat_of_coqfloat f)
```

Comment:
Why the change from `.15F` to `.18g` ?

Reply:
I suspect that %.15F is not precise enough -- I am using ocaml 4.05.0.
This can be seen by the following program:
#include <stdio.h>
#define PI 3.14159265358979323846
int main(){  printf("PI = %.15f\n",PI); }

The compiled C program and the dumped Clight program do not agree.
At Clight level, I get   printf(__stringlit_1, 3.14159265359);
And at runtime, PI = 3.141592653589793        vs     PI = 3.141592653590000

The problem vanishes with "%.18g" which is actually used by the x86 assembly printer.